apps/system/test/unit/accessibility_test.js

```diff
@@ -289,4 +313,67 @@ suite('system/Accessibility', function() {
         accessibility.quicknav.element.classList.contains('visible'));
     });
   });
+
+  suite('interaction hints', function() {
+    var screenNode, clock;
+    var realL10n = navigator.mozL10n;
+
+    setup(function() {
+      screenNode = document.createElement('div');
+      screenNode.id = 'screen';
+      document.body.appendChild(screenNode);
+
+      clock = this.sinon.useFakeTimers();
+
+      navigator.mozL10n = MockL10n; // jshint ignore:line
+    });
+
+    teardown(function() {
+      screenNode.parentNode.removeChild(screenNode);
+      clock.restore();
+      navigator.mozL10n = realL10n;
+    });
+
+    test('handle vc change event with hints enabled', function() {
+      var stubSetHintsTimeout = this.sinon.stub(accessibility,
+        'setHintsTimeout');
+      // This is where MozL10n was necessary in place of the stubbed
+      // accessibility.speak method
+      accessibility.handleAccessFuOutput(vcChangeHintsEnabledDetails);
```

Comment:
Just FYI -- The reason I included MockL10n in the first place was to be able to test whether setHintsTimeout would run after a vc-event fired with 'options.hints' populated.  I felt this was better than just seeing if accessibility.speak would be called with a known list of arguments.  Again, let me know if you'd prefer I did something else here.